schema/schema.v0.json

```diff
@@ -1456,6 +1910,87 @@
                     "default": false
                 }
             }
+        },
+        "Commands": {
```

Comment:
Where is this used?

This is a new definition but i cant find where its used.

Reply:
Oops. That was me prototyping for the comments above. I didn't intend to commit that.

If we're keen to progress down that path, we could add a comment of 'Future Work' for now, or I can back that bit out?

Reply:
Lets take it out for now.

Apart from that, this PR LGTM!